<?xml version="1.0"?>
<launch>
  <!-- Sim parameters
  <param name="/landshark/execution_monitor/max_anomaly" value = "0.95"/>
  <param name="/landshark/execution_monitor/obs_variance" value="0.15"/>
  <param name="/landshark/execution_monitor/acceptable_deviation" value="0.005"/>
  -->

  <!-- Robot parameters
  -->
  <param name="/landshark/cmu_z_test_monitor/max_anomaly" value = "0.95"/>
  <param name="/landshark/cmu_z_test_monitor/obs_variance" value="0.3"/>
  <param name="/landshark/cmu_z_test_monitor/acceptable_deviation" value="0.1"/>


  <param name="/landshark/cmu_z_test_monitor/play_alarm_sound" value="false"/>


  <node pkg="cmu_gps_drift" type="gps_drift_server" name="gps_drift_server" >
  </node>


  <param name="state_control_topic" value="/landshark/cmu_z_test_monitor/control" />
  <param name="state_status_topic" value="/landshark/cmu_z_test_monitor/status" />

  <node pkg="cmu_z_test_monitor" type="cmu_z_test_monitor_node" name="cmu_z_test_monitor_node" output="screen">
  </node>

  <node pkg="cmu_z_test_monitor" type="cmu_monitor_proxy" name="cmu_monitor_proxy" />

</launch> 

